Creating review instructions for tasks (#236)
kottans/frontend: pr-review-instructions-main-course.md

   * post links to your app on GitHub Pages and to your task PR in the [FE Questionarium chat](https://t.me/joinchat/DmX0JAl-mh5W0jrWli8Ycw) and ask mentors for a code review

### For peer-to-peer course:

   * publish your task code base for review following instructions in [FE P2P course homeworks repo](https://github.com/kottans/frontend-2019-p2p)
   * provide a review for this same task to at least 2 PR's of other students of p2p course
   * be reviewed by at least of 2 other students. For this, you can post links to your app on GitHub Pages and to your task PR in the [FE Students | Kottans](https://t.me/joinchat/DmX0JBHVkEhV1us2HdMmpA) and ask for a code review
   * once the code review phase successfully finished and you have approval from other students, post a message in [FE Questionarium chat](https://t.me/joinchat/DmX0JAl-mh5W0jrWli8Ycw):
   `{ Here must be the name of your task } — #p2p-pr-done` and add the link to your PR. **This step is important, as it helps mentors to track your progress!**
   
**When creating a PR do not forget to add a link to your app published on GitHub Pages**
      
# After finishing with a code review, do the following:

> * once the code review phase successfully finished and you have approval from one of the mentors add to your `kottans-frontend/README.md`:

Reviewer: ```suggestion
* once the code review phase successfully finished and you have required approvals add the following to your `kottans-frontend/README.md`:
```
